File: Triangles/RevRightTriangle/solidRevRightTriangle/solidRevRightTriangle.c
#include<stdio.h>

void main() {
    int numRows = 7; 
    int rowVal;     
    int colVal;     
    char patternSymbol = '*';   

    for(rowVal = numRows; rowVal >= 1; rowVal--) {
        for(colVal = 1 ;colVal <= numRows - rowVal; colVal++) {
            printf("  ");
        } 

        for(colVal = 1; colVal <= rowVal; colVal++) {
            printf("%c ", patternSymbol);
        }
        printf("\n");
    }
}

File: Star Patterns/Triangles/RevRightTriangle/solidRevRightTriangle/solidRevRightTriangle.py
numRows = 7
patternSymbol = '*'

for rowVal in range(numRows, 0, -1): #It will exclude 0 (Zero)
    for colVal in range(1, (numRows - rowVal) + 1):
        print("  ", end="")
    for colVal in range(1, rowVal + 1):
        print(patternSymbol, "", end="") # heer, I am only using this "" because, sep parameter is already giving us one space as sep=" "
    print()   # for new line
